docs/modules/plugins/pages/plugin-visual.adoc

```diff
@@ -267,6 +267,27 @@ When I <action> baseline with `scrollable-element-context` using screenshot conf
 |51               |SIMPLE          |
 ----
 
+==== Strategies
+
+[cols="1,3", options="header"]
+|===
+
+|Name
+|Description
+
+|`SIMPLE`
+|Used to take a screenshot of current viewport.
+
+|`FULL`
```

Comment:
Is this the right place for this? It's only available for mobile devices.
Also worth mentioning which strategy is the default one

Reply:
- This is under Mobile Native Applications section, so I think this is right place
- Will add